src/dfeed/web/web/part/post.d

```diff
@@ -237,7 +237,7 @@ void formatPost(Rfc850Post post, Rfc850Post[string] knownPosts, bool markAsRead
 			`<tr>` ~
 				`<td class="post-info">` ~
 					`<div class="post-author">`), html.putEncodedEntities(author), html.put(`</div>`);
-		putGravatar(gravatarHash, "http://www.gravatar.com/" ~ gravatarHash, `title="` ~ encodeHtmlEntities(_!`%s's Gravatar profile"`.format(author)), 80);
+		putGravatar(gravatarHash, "http://www.gravatar.com/" ~ gravatarHash, _!`%s's Gravatar profile`.format(author), null, 80);
```

Comment:
Not quite correct, `title` describes the link target but `alt` describes the image. The image alt text could just be `%s's Gravatar` and the title text should probably remain unchanged.

Reply:
oh there is supposed to be a link? Clicking the images doesn't actually do anything when on the forum (only focuses the message)

Reply:
Yep, the `img` tag is in an `a` tag.

If clicking doesn't result in navigating to the Gravatar page, that's a bug. We shouldn't be capturing that event.

In any case middle clicking probably should "work".